SoObjects/SOGo/SOGoContentObject.m

```diff
@@ -246,14 +246,12 @@ - (NSException *) saveComponent: (id) theComponent
 
 - (NSException *) copyToFolder: (SOGoGCSFolder *) newFolder
 {
-  [self subclassResponsibility: _cmd];
-
-  return nil;
+  return [self subclassResponsibility: _cmd];
```

Comment:
This change is not necessary as the return command is never executed because `[NSObject:subclassResponsability] raises a `NSException`.